Resources/migration.yml

```diff
@@ -537,3 +537,8 @@ LightTree03: LightTree
 LightTree04: LightTree
 LightTree05: LightTree
 LightTree06: LightTree
+
+#2024-12-28
+DrinkIrishCarBomb: DrinkIrishSlammer
+IrishCarBomb: IrishSlammer
+irishcarbomb: irishslammer
```

Comment:
the migrations only work for entity prototypes
see the comment on top of the file

Reply:
damn, lzk sniped the review

Reply:
Yeah, read one thing, brain did something completely different there.

Reply:
Thanks to both of you for walking me through it.